src/oncify.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Returns a function that runs only once
+ *
+ * @param {Function}  fn    the function to be made to run only once
+ *
+ * @returns {Function}      the function that runs only once
+ */
+const oncify = function (fn: Function): Function {
```

Comment:
Can this be renamed to `runOnce` so it's easy to remember?

`Oncify` doesn't really stick. What do you think?
I'm fine with leaving it as it is if you disagree though, it's not a hard thing lol.

Reply:
`runOnce` was the name I gave to it initially. I'll make this change